app/styles/play/menu/inventory-modal.sass

```diff
@@ -451,6 +451,9 @@ $itemSlotGridHeight: 51px
     
   #choose-hero-button
     left: 20px
```

Comment:
Update size for the new text length. Ideally would have reactive button lengths to the text content.